src/app/services/chart.service.ts

```diff
@@ -1521,6 +1519,35 @@ export class ChartService {
         laneChart.select('.y-axis')
           .selectAll('.tick text')
           .call(textWrap, margin.left);
+
+        //MIN and MAX labels
+        //--------------------
+
```

Comment:
Had to make some involved changes to the d3 code to get the min and max label to show up on the lane chart